tests/events.c

```diff
@@ -482,6 +494,20 @@ static void joystick_callback(int jid, int event)
     }
 }
 
+static void touch_callback(GLFWwindow* window, GLFWtouch* touchPoints, int count)
+{
+	printf("Priting info about all touch points");
```

Comment:
Sorry for these flaws. Is it preferred that I fix these things or do any of you desire to do it yourself?